modules/ROOT/pages/deploy-to-runtime-fabric.adoc

```diff
@@ -45,6 +45,11 @@ This tab displays the name and type of the deployment target you selected above.
 A minimum of two replicas is required for high availability.
 . To enable Mule clustering across each replica of the application, enable *Run in Runtime Clustering Mode*.
 
+** To enforce the deployment of replicas across different nodes, select *Enforce deploying replicas across nodes*. This option is only available for Runtime Fabric version 1.5 or later.
```

Comment:
It will be available for the upcoming release. Let's update the patch version once we confirm the release version along with the release doc.

Reply:
Updated.